.github/workflows/fips.yml

```diff
@@ -0,0 +1,135 @@
+name: CI FIPS
+
+on:
+  push:
+    branches: [master]
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
+  pull_request:
```

Comment:
I don't think we need to run this CI every PR. Can be triggered when a new version is released according to a specific PR title, such as:  `feat:release xxx`

Reply:
@spacewander @membphis So do I need to change it according to @soulbird ?